listenbrainz/db/stats.py

```diff
@@ -65,29 +65,27 @@ def insert_user_stats(user_id, artists, recordings, releases, artist_count):
         },
     }
 
-
     release_stats = {
         'all_time': {
             'releases': releases,
         }
     }
 
-
     with db.engine.connect() as connection:
         connection.execute(sqlalchemy.text("""
             INSERT INTO statistics.user (user_id, artist, recording, release)
                  VALUES (:user_id, :artists, :recordings, :releases)
             ON CONFLICT (user_id)
-          DO UPDATE SET artist = :artists,
+          DO UPDATE SET artist = statistics.user.artist || :artists,
```

Comment:
nice use of the concatenate operator, can we add tests to make sure that this works as we expect?